Neos.Flow/Classes/Package/GenericPackage.php

```diff
@@ -0,0 +1,321 @@
+<?php
+namespace Neos\Flow\Package;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Composer\ComposerUtility;
+use Neos\Flow\Core\ClassLoader;
+use Neos\Utility\Files;
+
+/**
+ * The generic base package that represents third party packages
+ */
+class GenericPackage implements PackageInterface, PackageKeyAwareInterface
```

Comment:
I'm not so sure about the name `GenericPackage` to be honest.. But `ComposerPackage` is not better, is it?
Then again, this is pretty low-level so it doesn't _really_ matter I guess

Reply:
I was almost creating `ComposerPackage` but then I would want to introdce something in between as well, because a composer package shouldn't know about package keys IMHO.